### PR TITLE
Translate style guide - priority B

### PR DESCRIPTION
**Pull request by Tintef**

### CODE REVIEW

File: src/v2/style-guide/index.md [hunk 1]
```diff
@@ -643,16 +643,13 @@ var myGreatMixin = {
...
 
+**Siempre que un compilador pueda concatenar archivos, cada componente debería estar en su propio archivo.**
 
-
-### Component files <sup data-p="b">strongly recommended</sup>
-
-**Whenever a build system is available to concatenate files, each component should be in its own file.**
-
-This helps you to more quickly find a component when you need to edit it or review how to use it.
+Esto lo ayudará a usted a encontrar de una manera más rápida una componente cuando precise editarlo o verificar como se utiliza.
```
Comment (sxcamacho) [inline src/v2/style-guide/index.md]: Typo `una componente` change to `un componente`

File: src/v2/style-guide/index.md [hunk 2]
```diff
@@ -686,11 +683,11 @@ components/
 
 
 
-### Single-file component filename casing <sup data-p="b">strongly recommended</sup>
+### Notación de nombres de componentes single-file <sup data-p="b">altamente recomendado</sup>
 
-**Filenames of [single-file components](../guide/single-file-components.html) should either be always PascalCase or always kebab-case.**
+**Los nombres de los archivos de las [componentes single-file](../guide/single-file-components.html) deben ser siempre PascalCase o siempre kebab-case.**
```
Comment (sxcamacho) [inline src/v2/style-guide/index.md]: Typo `de las componentes` change to `de los`

File: src/v2/style-guide/index.md [hunk 3]
```diff
@@ -722,9 +719,9 @@ components/
 
 
 
-### Base component names <sup data-p="b">strongly recommended</sup>
+### Nombre de componentes base <sup data-p="b">altamente recomendado</sup>
 
-**Base components (a.k.a. presentational, dumb, or pure components) that apply app-specific styling and conventions should all begin with a specific prefix, such as `Base`, `App`, or `V`.**
+**Las componentes base (también conocidas como componentes de presentación, "tontos", o puros) que aplican estilos y convenciones específicas de la aplicación, deben comenzar con un prefijo específico, tal como `Base`, `App` o `V`.**
```
Comment (sxcamacho) [inline src/v2/style-guide/index.md]: Typo `las componentes`

File: src/v2/style-guide/index.md [hunk 4]
```diff
@@ -733,23 +730,23 @@ components/
 </summary>
 {% endraw %}
 
-These components lay the foundation for consistent styling and behavior in your application. They may **only** contain:
+Estas componentes son la base para la definición de estilos y comportamientos consistentes en la aplicación. Estas **solamente** deben contener:
```
Comment (sxcamacho) [inline src/v2/style-guide/index.md]: - `Estas componentes` change to `Estos componentes`
- `Estas **solamente**` change to `Estos **solamente**`

File: src/v2/style-guide/index.md [hunk 5]
```diff
@@ -733,23 +730,23 @@ components/
...
 
-- HTML elements,
-- other base components, and
-- 3rd-party UI components.
+- elementos HTML,
+- otras componentes base, y
```
Comment (sxcamacho) [inline src/v2/style-guide/index.md]: Typo `otras componentes` change to `otros componentes`

File: src/v2/style-guide/index.md [hunk 6]
```diff
@@ -733,23 +730,23 @@ components/
...
+- elementos UI de terceros.
 
-But they'll **never** contain global state (e.g. from a Vuex store).
+Pero, **nunca** deben contener estado global (como una _store_ Vuex por ejemplo).
 
-Their names often include the name of an element they wrap (e.g. `BaseButton`, `BaseTable`), unless no element exists for their specific purpose (e.g. `BaseIcon`). If you build similar components for a more specific context, they will almost always consume these components (e.g. `BaseButton` may be used in `ButtonSubmit`).
+Sus nombres generalmente incluyen el nombre del elemento que envuelven(por ejemplo `BaseButton`, `BaseTable`), a menos que no exista un elemento para su propósito específico (por ejemplo `BaseIcon`). Si usted construye componentes similares para un contexto más específico, ellas casi siempre consumirán estas componentes (por ejemplo `BasButton` podría ser consumido en `ButtonSubmit`).
```
Comment (sxcamacho) [inline src/v2/style-guide/index.md]: - Please, add a space at `envuelven (por ejemplo`
- Plase, change `ellas casi siempre consumirán estas componentes` to `casi siempre consumirán estos componentes` or something like that.

File: src/v2/style-guide/index.md [hunk 7]
```diff
@@ -686,11 +683,11 @@ components/
...
 
 
-### Single-file component filename casing <sup data-p="b">strongly recommended</sup>
+### Notación de nombres de componentes single-file <sup data-p="b">altamente recomendado</sup>
 
-**Filenames of [single-file components](../guide/single-file-components.html) should either be always PascalCase or always kebab-case.**
+**Los nombres de los archivos de los [componentes single-file](../guide/single-file-components.html) deben ser siempre PascalCase o siempre kebab-case.**
 
-PascalCase works best with autocompletion in code editors, as it's consistent with how we reference components in JS(X) and templates, wherever possible. However, mixed case filenames can sometimes create issues on case-insensitive file systems, which is why kebab-case is also perfectly acceptable.
+El autocompletado de los editores de código funciona mejor cuando se utiliza PascalCase, ya que esta es consistente con la forma en que referenciamos componenten en JS(X) y _templates_, dónde sea posible. Sin embargo, nombres de archivos mixtos pueden crear problemas en sistemas de archivos insensibles a las mayúsculas y minúsculas, es por esto que utilizar kebab-case es perfectamente aceptable.
```
Comment (FLarra) [inline src/v2/style-guide/index.md]: Minor suggestion ?)
```suggestion
El autocompletado de los editores de código funciona mejor cuando se utiliza PascalCase, ya que esta es consistente con la forma en que referenciamos componenten en JS(X) y plantillas, dónde sea posible. Sin embargo, nombres de archivos mixtos pueden crear problemas en sistemas de archivos insensibles a las mayúsculas y minúsculas, es por esto que utilizar kebab-case es perfectamente aceptable.
```

File: src/v2/style-guide/index.md [hunk 8]
```diff
@@ -733,23 +730,23 @@ components/
...
+Pero, **nunca** deben contener estado global (como una _store_ Vuex por ejemplo).
 
-Their names often include the name of an element they wrap (e.g. `BaseButton`, `BaseTable`), unless no element exists for their specific purpose (e.g. `BaseIcon`). If you build similar components for a more specific context, they will almost always consume these components (e.g. `BaseButton` may be used in `ButtonSubmit`).
+Sus nombres generalmente incluyen el nombre del elemento que envuelven (por ejemplo `BaseButton`, `BaseTable`), a menos que no exista un elemento para su propósito específico (por ejemplo `BaseIcon`). Si usted construye componentes similares para un contexto más específico, casi siempre consumirán estos componentes (por ejemplo `BaseButton` podría ser consumido en `ButtonSubmit`).
 
-Some advantages of this convention:
+Algunas ventajas de esta convención:
 
-- When organized alphabetically in editors, your app's base components are all listed together, making them easier to identify.
+- Cuando están organizados alfabéticamente en los editores, los componentes base serán listados todos juntos, volviendose más fácil de identificar.
```
Comment (FLarra) [inline src/v2/style-guide/index.md]: Small misspelling  here.

```suggestion
- Cuando están organizados alfabéticamente en los editores, los componentes base serán listados todos juntos, volviéndose más fácil de identificar.
```

File: src/v2/style-guide/index.md [hunk 9]
```diff
@@ -805,11 +802,11 @@ components/
 
 
 
-### Single-instance component names <sup data-p="b">strongly recommended</sup>
+### Nombres de componentes de instancia úncia <sup data-p="b">altamente recomendado</sup>
```
Comment (FLarra) [inline src/v2/style-guide/index.md]: Typo here: 

```suggestion
### Nombres de componentes de instancia única <sup data-p="b">altamente recomendado</sup>
```

File: src/v2/style-guide/index.md [hunk 10]
```diff
@@ -805,11 +802,11 @@ components/
...
 
-**Components that should only ever have a single active instance should begin with the `The` prefix, to denote that there can be only one.**
+**Componentes que deben tener solamente una única instancia activa deben comenzar con el prefijo `The`, para denotar que solo puede haber una.**
 
-This does not mean the component is only used in a single page, but it will only be used once _per page_. These components never accept any props, since they are specific to your app, not their context within your app. If you find the need to add props, it's a good indication that this is actually a reusable component that is only used once per page _for now_.
+Esto no quiere decir que la componente solamente es utiliza en una única página, sino que solamente será utilizada una vez _por página_. Este tipo de componente nunca aceptan _props_, dado que son específicas para la aplicación, no del contexto de la misma. Si usted encuentra la necesidad de añadir _props_, puede ser un buen indicio de que se trata de un comoponente reusable que solamente es utiliza una vez por página, _por ahora_.
```
Comment (FLarra) [inline src/v2/style-guide/index.md]: A couple of mistakes:

```suggestion
Esto no quiere decir que el componente solamente se utiliza en una única página, sino que solamente será utilizado una vez _por página_. Este tipo de componente nunca aceptan propiedades, dado que son específicas para la aplicación, no del contexto de la misma. Si usted encuentra la necesidad de añadir propiedades, puede ser un buen indicio de que se trata de un componente reusable que solamente se utiliza una vez por página, _por ahora_.
```

File: src/v2/style-guide/index.md [hunk 11]
```diff
@@ -868,10 +865,10 @@ components/
 |- TodoList.vue
 ```
 
-This isn't recommended, as it results in:
+Esto no es recomendado, ya que resulta en:
 
-- Many files with similar names, making rapid file switching in code editors more difficult.
-- Many nested sub-directories, which increases the time it takes to browse components in an editor's sidebar.
+- Muchas archivos con nombres similares, haciendo difícil cambiar entre archivos en un editor de código.
```
Comment (FLarra) [inline src/v2/style-guide/index.md]:
```suggestion
- Muchos archivos con nombres similares, haciendo difícil cambiar entre archivos en un editor de código.
```

File: src/v2/style-guide/index.md [hunk 12]
```diff
@@ -868,10 +865,10 @@ components/
...
+- Muchas subdirectorios anidados, lo cual incremente el tiempo que toma buscar un componente específico en un editor.
```
Comment (FLarra) [inline src/v2/style-guide/index.md]:
```suggestion
- Muchos subdirectorios anidados, lo cual incrementa el tiempo que toma buscar un componente específico en un editor.
```

File: src/v2/style-guide/index.md [hunk 13]
```diff
@@ -922,19 +919,20 @@ components/
...
 {% endraw %}
 
-You may be wondering:
+Usted se debe estar preguntando:
 
-> "Why would we force component names to use less natural language?"
+> "¿Por qué forzar un lenguaje menos natural al nombrar componentes?"
 
-In natural English, adjectives and other descriptors do typically appear before the nouns, while exceptions require connector words. For example:
+En el inglés natural, adjetivos y otros descriptores aparecen tipicamente antes de los sustantivos, mientras que excepciones requiren conectores. Por ejemplo:
```
Comment (FLarra) [inline src/v2/style-guide/index.md]: Maybe :?
```suggestion
En inglés, adjetivos y otros descriptores aparecen típicamente antes de los sustantivos, mientras que excepciones requieren conectores. Por ejemplo:
```

File: src/v2/style-guide/index.md [hunk 14]
```diff
@@ -922,19 +919,20 @@ components/
...
+- Café _con_ leche
+- Sopa _del_ día
+- Visitante _del_ museo
 
-You can definitely include these connector words in component names if you'd like, but the order is still important.
+Usted puede definitivamente incluir estos conectores en el nombre de los componentes si así lo desea, pero el orden de las palabras sigue siendo importante.
 
-Also note that **what's considered "highest-level" will be contextual to your app**. For example, imagine an app with a search form. It may include components like this one:
+
+También observe que **lo qué es considerado "de más alto nivle" será contextual a su aplicación**. Por ejemplo, imagine una aplicación con un formulario de búsqueda. Este puede incluir componentes como:
```
Comment (FLarra) [inline src/v2/style-guide/index.md]: Typo: 

```suggestion
También observe que **lo que es considerado "de más alto nivel" será contextual a su aplicación**. Por ejemplo, imagine una aplicación con un formulario de búsqueda. Este puede incluir componentes como:
```

File: src/v2/style-guide/index.md [hunk 15]
```diff
@@ -998,24 +996,24 @@ components/
...
+### Componentes con cierre automático <sup data-p="b">altamente recomendado</sup>
 
-**Components with no content should be self-closing in [single-file components](../guide/single-file-components.html), string templates, and [JSX](../guide/render-function.html#JSX) - but never in DOM templates.**
+**Componentes sin contenido deben cerrarse automáticamenete en [componentes single-file](../guide/single-file-components.html), _templates_ basados en _strings_, y [JSX](../guide/render-function.html#JSX) - pero nunca en _DOM templates_.**
 
-Components that self-close communicate that they not only have no content, but are **meant** to have no content. It's the difference between a blank page in a book and one labeled "This page intentionally left blank." Your code is also cleaner without the unnecessary closing tag.
+Los componentes que se cierran automáticamente no solo comunican que no tienen contenido, sino que garantizan que no deben tener contendio. Es la diferencia entre una página en blanco en un libro y una con el texto "Está página fue intencionalmente dejada en blanco". También, su código es más limpio sin la _tag_ de cerrado innecesaria.
```
Comment (FLarra) [inline src/v2/style-guide/index.md]: Typo:

```suggestion
Los componentes que se cierran automáticamente no solo comunican que no tienen contenido, sino que garantizan que no deben tener contenido. Es la diferencia entre una página en blanco en un libro y una con el texto "Está página fue intencionalmente dejada en blanco". También, su código es mas limpio sin la _tag_ de cerrado innecesaria.
```

File: src/v2/style-guide/index.md [hunk 16]
```diff
@@ -998,24 +996,24 @@ components/
...
 
 ``` html
-<!-- In single-file components, string templates, and JSX -->
+<!-- En componentes single-file, templates basados en string, y JSX -->
 <MyComponent></MyComponent>
 ```
 
 ``` html
-<!-- In DOM templates -->
+<!-- En DOM templates -->
```
Comment (FLarra) [inline src/v2/style-guide/index.md]:
```suggestion
<!-- En plantillas del DOM -->
```

File: src/v2/style-guide/index.md [hunk 17]
```diff
@@ -998,24 +996,24 @@ components/
...
 
-Unfortunately, HTML doesn't allow custom elements to be self-closing - only [official "void" elements](https://www.w3.org/TR/html/syntax.html#void-elements). That's why the strategy is only possible when Vue's template compiler can reach the template before the DOM, then serve the DOM spec-compliant HTML.
+Desafortunadamente, HTML no permite que los elementos personalizados se cierren automáticamente - sólo los [official "void" elements](https://www.w3.org/TR/html/syntax.html#void-elements). Es por eso que esta estrategia solo es posible cuando los _templates_ Vue son compilados antes de estar en el DOM, por lo que pueden servir HTML compatible con la especifiación.
 
 {% raw %}<div class="style-example example-bad">{% endraw %}
 #### Incorrecto
 
 ``` html
-<!-- In single-file components, string templates, and JSX -->
+<!-- En componentes single-file, templates basados en string, y JSX -->
```
Comment (FLarra) [inline src/v2/style-guide/index.md]:
```suggestion
<!-- En componentes de un solo archivo, plantillas basadas en cadenas de texto, y JSX -->
```

File: src/v2/style-guide/index.md [hunk 18]
```diff
@@ -1024,47 +1022,47 @@ Unfortunately, HTML doesn't allow custom elements to be self-closing - only [off
 #### Correcto
 
 ``` html
-<!-- In single-file components, string templates, and JSX -->
+<!-- En componentes single-file, templates basados en string, y JSX -->
```
Comment (FLarra) [inline src/v2/style-guide/index.md]: Same here.

File: src/v2/style-guide/index.md [hunk 19]
```diff
@@ -1024,47 +1022,47 @@ Unfortunately, HTML doesn't allow custom elements to be self-closing - only [off
...
 <MyComponent/>
 ```
 
 ``` html
-<!-- In DOM templates -->
+<!-- En DOM templates -->
```
Comment (FLarra) [inline src/v2/style-guide/index.md]: The same suggestion than above here :)

File: src/v2/style-guide/index.md [hunk 20]
```diff
@@ -1024,47 +1022,47 @@ Unfortunately, HTML doesn't allow custom elements to be self-closing - only [off
...
+**En la mayoría de los proyectos, los nombres de los componentes deben ser siempre PascalCase en [componentes single-file](../guide/single-file-components.html) y _templates_ basados en _string_ - pero kebab-case en DOM _templates_.**
 
-PascalCase has a few advantages over kebab-case:
+PascalCase tiene algunas ventajas sobre kebab-case:
 
-- Editors can autocomplete component names in templates, because PascalCase is also used in JavaScript.
-- `<MyComponent>` is more visually distinct from a single-word HTML element than `<my-component>`, because there are two character differences (the two capitals), rather than just one (a hyphen).
-- If you use any non-Vue custom elements in your templates, such as a web component, PascalCase ensures that your Vue components remain distinctly visible.
+- Editores puede autocompletar nombres de componentes en _templates_, ya que en JavaScript también se utiliza PascalCase.
+- `<MyComponent>` es más disntivo visualmente que un elemento HTML simple que `<my-component>`, porque hay dos caracteres distintos (las dos mayúsculas), en lugar de solo uno (el guión).
```
Comment (FLarra) [inline src/v2/style-guide/index.md]:
```suggestion
- `<MyComponent>` es más distintivo visualmente que un elemento HTML simple que `<my-component>`, porque hay dos caracteres distintos (las dos mayúsculas), en lugar de solo uno (el guión).
```

File: src/v2/style-guide/index.md [hunk 21]
```diff
@@ -1024,47 +1022,47 @@ Unfortunately, HTML doesn't allow custom elements to be self-closing - only [off
...
+- Si usted utiliza cualquier elemento no Vue en sus _templates_, como un componente web, PascalCase asegura que sus componente Vue se mantendrán distinguibles visualmente.
```
Comment (FLarra) [inline src/v2/style-guide/index.md]:
```suggestion
- Si usted utiliza cualquier elemento no Vue en sus plantillas, como un componente web, PascalCase asegura que sus componente Vue se mantendrán distinguibles visualmente.
```

**Review by FLarra — CHANGES_REQUESTED** [review]

File: src/v2/style-guide/index.md (hunk 15 above)
Comment (Tintef) [reply src/v2/style-guide/index.md]: @FLarra That `más` must have an accent

**Review by sxcamacho — APPROVED** [review]

**Pull request merged**